lib/drivers/veres.js

```diff
@@ -636,13 +640,21 @@ api.info = async options => {
   async function looper(result) {
     // omit "not found" info if requested
     if(!options.retryShowFound || (options.retryShowFound && result.found)) {
-      _info(result, options);
+      if(callback) {
```

Comment:
We favor the 'return early' strategy as opposed to if/else when possible.  So this becomes:
```
if(callback) {
  return callback(...);
}

Reply:
Agreed, that would be preferable. The reason there's no return there is that the _info() display logic is cumulative, somehow interacts with multiple retries below.

Reply:
will this result in the callback being called more than once in some circumstances?

Reply:
@mattcollier - Hmm possibly. I think the idea is that it'll only get called more than once with a retry flag, and downstream caller code probably shouldn't use that.